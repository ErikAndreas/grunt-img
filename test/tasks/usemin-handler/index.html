<!DOCTYPE html>
<html>
    <head>
        <title>Grunt test</title>
        <!-- build:css css/app.css -->
        <link rel="stylesheet" href="css/app-1.css">
        <link rel="stylesheet" href="css/app-2.css">
        <!-- endbuild -->

        <!-- build:js js/head.js -->
        <script src="js/head-1.js"></script>
        <script src="js/head-2.js"></script>
        <!-- endbuild -->
    </head>
    <body>

        <!-- build:js js/app.js -->
        <script src="js/app-1.js"></script>
        <script src="js/app-2.js"></script>
        <script src="js/app-3.js"></script>
        <script src="js/app-4.js"></script>
        <script src="js/app-5.js"></script>
        <!-- endbuild -->
    </body>
</html>