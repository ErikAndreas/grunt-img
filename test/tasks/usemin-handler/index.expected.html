<!DOCTYPE html>
<html>
    <head>
        <title>Grunt test</title>
        <link rel="stylesheet" href="css/07d0750b.app.css">

        <script src="js/ad0a1fd9.head.js"></script>
    </head>
    <body>

        <script src="js/7d1bdffe.app.js"></script>
    </body>
</html>